builder/builder_utilities.livecodescript

```diff
@@ -547,6 +552,35 @@ end macPList
 
 ////////////////////////////////////////////////////////////////////////////////
 
+command builderEnsureZip pZipfile, pURL
+   if there is not a file pZipfile then
+      builderLog "message", "Downloading zip file:" && pZipfile
+      get shell(merge("curl -o '[[pPath]]' '[[pUrl]]'"))
+      if the result is not zero then
```

Comment:
Yes it should! Thanks for spotting that... Looking forward to implementing some basic refactoring in the script editor.